pvlib/scaling.py

```diff
@@ -0,0 +1,320 @@
+"""
+The ``scaling`` module contains functions for manipulating irradiance
+or other variables to account for temporal or spatial characteristics.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def wvm(cs_series, latitude, longitude, cloud_speed,
```

Comment:
`latitude` and `longitude` are converted to Northing/Easting coordinates, in order to compute distances. I think (but am not sure) that it would be better to input Northing/Easting to `wvm` and supply a public helper function that converts a list of (lat,long) pairs to a list of (Northing, Easting) pairs.

I'm more sure (but not certain) that a single point `points` as a list of tuples (northing, easting) is better, This would match how geopandas and shapely store coordinates. In the longer term (not in this PR, I am certain) we could consider extend this interface to accept `shapely` or `geopandas` files as input.

Would appreciate other's views here @wholmgren @mikofski

Reply:
Thinking a bit more about this, the wavelet transform is agnostic about direction - it only cares about distance. So there's no need to maintain any particular orientation for the x,y coordinate axes. The input could be a list of (x, y) coordinate pairs dimensioned in meters.

Reply:
Thanks @jranalli for this contribution. I agree with @cwhanse, a list of tuples makes more sense to me than the original `pvl_WVM` API of separate latitude and longitude lists. Whether these are `(lat, lon)` pairs or `(x_east, y_north)` I'm less sure of.

I did find these existing conversion methods:
- [convertbng](https://pypi.org/project/convertbng/)
- [`pyproj.transform()`](https://pyproj4.github.io/pyproj/dev/api/transformer.html#pyproj-transform)
- [blog demo of`pyproj.transform()`](https://altons.github.io/python/2012/12/01/converting-northing-and-easting-to-latitude-and-longitude/)

So it does seem like`Point` syntax is more common. @anomam has experience using Shapely and GeoPandas.